### PR TITLE
change readme.md to use updated engine filename

### PR DESCRIPTION
**Pull request by ElleryW**

### CODE REVIEW

**Comment by nikukyugamer** [comment]
`stable_diffusion_engine.py` is nothing more than class file so `stable_diffusion.py` should be replaced to `demo.py`.

File: README.md [hunk 1]
```diff
@@ -51,17 +51,17 @@ optional arguments:
 
 ### Example Text-To-Image
 ```bash
-python stable_diffusion.py --prompt "Street-art painting of Emilia Clarke in style of Banksy, photorealism"
+python stable_diffusion_engine.py --prompt "Street-art painting of Emilia Clarke in style of Banksy, photorealism"
```
Comment (nikukyugamer) [inline README.md:54]:
```suggestion
python demo.py --prompt "Street-art painting of Emilia Clarke in style of Banksy, photorealism"
```

File: README.md [hunk 2]
```diff
@@ -51,17 +51,17 @@ optional arguments:
...
 ```
 
 ### Example Image-To-Image
 ```bash
-python stable_diffusion.py --prompt "Photo of Emilia Clarke with a bright red hair" --init-image ./data/input.png --strength 0.5
+python stable_diffusion_engine.py --prompt "Photo of Emilia Clarke with a bright red hair" --init-image ./data/input.png --strength 0.5
```
Comment (nikukyugamer) [inline README.md:59]:
```suggestion
python demo.py --prompt "Photo of Emilia Clarke with a bright red hair" --init-image ./data/input.png --strength 0.5
```

File: README.md [hunk 3]
```diff
@@ -51,17 +51,17 @@ optional arguments:
...
 ```
 
 ### Example Inapinting
 ```bash
-python stable_diffusion.py --prompt "Photo of Emilia Clarke with a bright red hair" --init-image ./data/input.png --mask ./data/mask.png --strength 0.5
+python stable_diffusion_engine.py --prompt "Photo of Emilia Clarke with a bright red hair" --init-image ./data/input.png --mask ./data/mask.png --strength 0.5
```
Comment (nikukyugamer) [inline README.md:64]:
```suggestion
python demo.py --prompt "Photo of Emilia Clarke with a bright red hair" --init-image ./data/input.png --mask ./data/mask.png --strength 0.5
```

**Comment by bes-dev** [comment]
👍

**Pull request merged**